portfolio/src/main/webapp/script.js

```diff
@@ -45,26 +75,45 @@ function getComments() {
 
     console.log(commentsArray);
 
-    if (!(commentsArray && commentsArray.length)) return;
+    if (!(commentsArray && commentsArray.length)) {
+      removeCommentsColumn();
+      return;
+    }
 
-    const commentsOutput = document.getElementById("comments-output")
+    addCommentsColumn();
+
+    const commentsOutput = document.getElementById("comments_body")
       .appendChild(document.createElement("h2")
         .appendChild(document.createTextNode("Comments")).parentElement);
 
-    const cmtDOM = document.createElement("ul");
+    const cmtDOM = document.createElement("div");
+    let quoteDOM;
```

Comment:
Do you need to declare these variables outside the lambda you're passing to `forEach`?

Reply:
Is it better to initialized variables in a for loop if the for loop is their scope? 

I always initialized them outside to improve performance.

Reply:
In Javascript, I don't believe there should be any difference in performance (e.g., check out this StackOverflow [link](https://stackoverflow.com/questions/3684923/javascript-variables-declare-outside-or-inside-loop#:~:text=There%20is%20absolutely%20no%20difference,command%20executed%20at%20run%2Dtime.)). 

See 5.1.3 in the JS style guide (https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations). By declaring local variables using the `let` keyword inside the loop, you'll minimize their scope to just the loop body.

Reply:
Oh ok, I didn't know that about Javascript